app/services/message_employee_matcher.rb

```diff
@@ -14,4 +14,15 @@ def run
   def day_count
     message.days_after_start
   end
+
+  def match_time(time_zone)
+    employee_current_hour = Time.current.in_time_zone(time_zone).hour
+    employee_current_minute = Time.current.in_time_zone(time_zone).min
+
+    employee_current_hour == message.time_of_day.hour && employee_current_minute == message.time_of_day.min
+  end
+
+  def message_already_sent?(employee)
```

Comment:
Yes, we should add a spec!  I was a little unsure of how to do so, but that's a great suggestion.

I am also trying to keep in mind your conventions for how to name the methods ending in `?` with regards to testing because I think they are very helpful. :-)  I will take a closer look and see what will make more sense!